.github/workflows/coding-standards.yml

```diff
@@ -53,3 +56,61 @@ jobs:
       # The -q option is required until phpcs v4 is released
       - name: "Run PHP_CodeSniffer"
         run: "vendor/bin/phpcs -q --no-colors --report=checkstyle | cs2pr"
+
+      - name: "Format the code"
```

Comment:
Nice to have, removes the need to remember to run this locally

Reply:
I can see the benefit, but I'm not sure...

Reply:
Just an idea, I'm fine with removing it if need be

Reply:
I think formatting the code should happen after each commit but this might make things ugly.

Reply:
@divine keep it or remove it?